src/components/tools/ManipulatorTool.vue

```diff
@@ -38,38 +39,79 @@ export default defineComponent({
         : viewProxy.value.getInteractorStyle3D();
     });
 
-    const manipulator = props.manipulatorClass.newInstance(options.value);
+    const manipulators: any[] = [];
 
-    watch(options, (newOptions) => {
-      if ('button' in newOptions) {
-        manipulator.setButton(newOptions.button);
-      }
-      if ('shift' in newOptions) {
-        manipulator.setShift(newOptions.shift);
-      }
-      if ('control' in newOptions) {
-        manipulator.setControl(newOptions.control);
+    function updateStyle(
+      style: any,
+      newManipulators: any[],
+      oldManipulators: any[]
+    ) {
+      if (style !== null) {
+        oldManipulators.forEach((m) => {
+          if (m.isA('vtkCompositeMouseManipulator')) {
```

Comment:
I'm thinking we don't need the `m.isA('vtkCompositeMouseManipulator')` checks, since manipulators come from the `manipulatorClass`, which should be a mouse manipulator class. If the user passes in an invalid class, then that's on them.

Reply:
Ok, I can change that. Perhaps the file named `MouseManipulatorTool.vue` if we don't expect any other manipulators to be handled by it. Note that this PR is based on the unmerged PR 229, so I will make these changes in the other PR and rebase this one later directly to master.